### ISSUE TITLE
При заливке первой стратегии она не становится текущей автоматом.

### ISSUE THREAD

**Issue by Megabyteceer** [sat2707/aicups#28]

**Comment by 678098** [comment]
Возможно, это и не надо делать. Вдруг новая стратегия работает плохо? Вдруг она не протестирована против реальных сильных игроков (даже если появится локал раннер, всё равно доступа к чужим стратегиям для сравнения нет - силу стратегии можно познать только на тестовой площадке)?

**Comment by Megabyteceer** [comment]
Можно добавить предупреждение в случае если текущая стратегия не указана.

**Comment by sat2707** [comment]
Да, это будет полезно. При заливке первой стратегии она становится автоматом выбранной для дуэли, при успешной дуэли - автоматом выбранной для рейтинга.
